gef.py

```diff
@@ -8784,6 +8823,102 @@ def do_invoke(self, argv):
 
         return
 
+
+@register_command
+class HighlightCommand(GenericCommand):
+    """
+    This command highlights user defined text matches which modifies GEF output universally.
+    """
+    _cmdline_ = "highlight"
+    _syntax_ = "{} (add|del|list|clear|regex)".format(_cmdline_)
+    _aliases_ = ["hl"]
+
+    def __init__(self):
+        super(HighlightCommand, self).__init__(prefix=True)
+
+    def do_invoke(self, argv):
+        self.usage()
+
+
+@register_command
+class HighlightListCommand(GenericCommand):
+    """Show the current highlight table with matches to colors."""
+    _cmdline_ = "highlight list"
+    _aliases_ = ["highlight ls", "hll"]
+    _syntax_ = _cmdline_
+
+    def do_invoke(self, argv):
+        print_highlight_table()
+
+
+@register_command
+class HighlightClearCommand(GenericCommand):
+    """Clears the highlight table, removing all matches."""
+    _cmdline_ = "highlight clear"
+    _syntax_ = _cmdline_
+
+    def do_invoke(self, argv):
+        highlight_table.clear()
+
+
+@register_command
+class HighlightAddCommand(GenericCommand):
+    """Adds a match to the highlight table."""
+    _cmdline_ = "highlight add"
+    _syntax_ = "{} [match] [color]".format(_cmdline_)
```

Comment:
These aren't optional, so it should be MATCH COLOR.
(Square brackets indicate optional fields)